arcgis-ios-sdk-samples/Maps/Open map (URL)/OpenMapURLViewController.swift

```diff
@@ -35,12 +35,12 @@ class OpenMapURLViewController: UIViewController {
     }
     
     private let mapModels: [MapAtURL] = [
```

Comment:
I would consider renaming this to "mapOptions" or something similar. But it's not a big deal; up to you.